File: TopflytechCodec/Java/topflytechCodec/src/com/topflytech/codec/entities/OneWireData.java
package com.topflytech.codec.entities;

public class OneWireData {
    private byte deviceType;
    private String deviceId;
    private byte[] oneWireContent;

    public byte getDeviceType() {
        return deviceType;
    }

    public void setDeviceType(byte deviceType) {
        this.deviceType = deviceType;
    }

    public String getDeviceId() {
        return deviceId;
    }

    public void setDeviceId(String deviceId) {
        this.deviceId = deviceId;
    }

    public byte[] getOneWireContent() {
        return oneWireContent;
    }

    public void setOneWireContent(byte[] oneWireContent) {
        this.oneWireContent = oneWireContent;
    }
}

File: TopflytechCodec/C#/TopflytechCodec/TopflytechCodec/Entities/OneWireData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class OneWireData
    {
        private byte _deviceType;
        private string _deviceId;
        private byte[] _oneWireContent;

        public byte DeviceType
        {
            get => _deviceType;
            set => _deviceType = value;
        }

        public string DeviceId
        {
            get => _deviceId;
            set => _deviceId = value;
        }

        public byte[] OneWireContent
        {
            get => _oneWireContent;
            set => _oneWireContent = value;
        }
    }
}
